compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -8690,7 +8690,7 @@ TR::Node *idivSimplifier(TR::Node * node, TR::Block * block, TR::Simplifier * s)
                }
             node->getFirstChild()->incReferenceCount();
             }
-         else if (s->cg()->getSupportsLoweringConstIDiv() && !isPowerOf2(divisor) &&
+         else if (s->cg()->getSupportsLMulHigh() && !isPowerOf2(divisor) &&
```

Comment:
It is `getSupportsIMulHigh()`, isn't it?

Reply:
Yes!